gears/src/x/errors.rs

```diff
@@ -1,17 +1,23 @@
 use std::fmt::Display;
 
 use address::AccAddress;
+use cosmwasm_std::Uint256;
 use thiserror::Error;
 
 use crate::{
     application::handlers::node::{ErrorCode, TxError},
-    error::AppError,
     types::{
+        base::errors::CoinsError,
+        denom::Denom,
         gas::GasMeteringErrors,
         store::gas::errors::{GasStoreErrorKinds, GasStoreErrors},
     },
 };
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, thiserror::Error)]
+#[error("account does not exist")]
+pub struct AccountNotFound;
```

Comment:
We have a few types of accounts and I suggest that we can add data with string of byte address.

Reply:
Done